Add Product use case

manototh opened the pull request:
Preview:
https://axiom-mano-ai-product-use-cases.mintlify.app/getting-started-guide/product-analytics

tothmano committed:
Add AI and Product use cases

tothmano committed:
Update ai-machine-learning.mdx

dominicchapman commented on getting-started-guide/product-analytics.mdx:
```diff
@@ -0,0 +1,44 @@
+
+This page explains how you can leverage the power of event data for the product analytics use case.
+
+In product analytics, the ability to harness and interpret data effectively can determine the success of a product. Axiom offers a new approach to product analytics by leveraging the power of timestamped event data. This unique capability enables organizations to gain actionable insights, optimize user experiences, and drive product innovation.
```
I think it’s a stretch to call this new. Mixpanel, Amplitude and friends have been leading the charge for close to a decade. The part that feels novel to me is the ability to read every single event easily (sometimes they’ll make it easy to present aggregates, but not so easy to dive into specific underlying events), and I would expect performance wise we’d crush them at massive scale.

dominicchapman commented on getting-started-guide/product-analytics.mdx:
```diff
@@ -0,0 +1,44 @@
+
+The following key features make Axiom perfect for product analytics:
+
+- **Real-time event monitoring:** Axiom’s ability to ingest and process data in real-time means you can monitor user activity as it happens. This empowers product managers to act quickly in response to anomalies or unexpected usage patterns, reducing downtime and improving user satisfaction.
```
What kinds of anomalies or unexpected usage patterns are usually of interest to product managers?

dominicchapman commented on getting-started-guide/product-analytics.mdx:
```diff
@@ -0,0 +1,44 @@
+The following key features make Axiom perfect for product analytics:
+
+- **Real-time event monitoring:** Axiom’s ability to ingest and process data in real-time means you can monitor user activity as it happens. This empowers product managers to act quickly in response to anomalies or unexpected usage patterns, reducing downtime and improving user satisfaction.
+- **Unified data platform:** Axiom eliminates data silos by integrating event data from diverse sources into a single, cohesive platform. Whether your data originates from web applications, mobile apps, or third-party tools, Axiom ensures seamless access and analysis.
```
Yeah great point. Could go further: What does it unlock when system / infrastructure telemetry is in the same store as product analytics? That really is unusual compared to vendors like Mixpanel and Amplitude.

dominicchapman commented on getting-started-guide/product-analytics.mdx:
```diff
@@ -0,0 +1,44 @@
+- **Custom dashboards and visualizations:** Intuitive dashboards and customizable visualizations make it easy for product managers to communicate insights to stakeholders. Axiom’s visual tools enhance collaboration and decision-making.
+- **Scalable infrastructure:** As your product grows, so does the volume of event data. Axiom’s architecture is built to scale effortlessly, ensuring that your analytics remain robust and reliable, even with massive datasets.
+
+## Enhance product analytics with Axiom
```
While this section is surely relevant, unless we provide a working examples of relevant APL I’m not sure how much confidence this would give me as a product person considering Axiom

dominicchapman commented on getting-started-guide/product-analytics.mdx:
```diff
@@ -0,0 +1,44 @@
+sidebarTitle: Product analytics
+---
+
+Axiom helps you leverage the power of timestamped event data. Axiom believes that event data reflects a broad range of interactions, crossing boundaries from engineering to product management, security, and beyond. For a more general explanation of event data in Axiom, see [Events](/getting-started-guide/event-data).
```
I think it would be a shame to publish content on product analytics and not touch on what has become something of a standard in Segment’s spec https://segment.com/docs/connections/spec/ 

`Identify`, `track`, and `group` calls provide the rich foundation for event analytics, and it’s pretty common at this point for vendors to accept Segment events or services like RudderStack which follow the identical pattern because of it. Indeed, we ourselves use this in staging > `segment-frontend-prod` and `segment-core-prod` so you can see the schema there with `segment-frontendprod | getschema`. 

Examples:
https://docs.mixpanel.com/docs/tracking-methods/integrations/segment
https://amplitude.com/docs/data/source-catalog/segment
https://help.june.so/en/articles/5988266-segment-setup

If nothing else, it would give a fairly approachable foundation in this material to then provide sample APL that is directionally sound for product folks. 

Excited about this content - I think this could be a big win!

dominicchapman commented on getting-started-guide/ai-machine-learning.mdx:
Can you omit this for now. For... reasons..

tothmano committed:
First fixes

tothmano committed:
Merge branch 'main' into mano/ai-product-use-cases

tothmano committed:
Update learn-about-axiom.mdx

manototh committed:
Merge branch 'main' into mano/ai-product-use-cases

tothmano committed:
Address feedback

manototh commented:
@dominicchapman Many thanks for the detailed feedback. I've addressed these issues in https://github.com/axiomhq/docs/pull/290/commits/ecbfc758c90c3aa4ba09949240b3dbc26fef93e9

tothmano committed:
Implement feedback

manototh committed:
Merge branch 'main' into mano/ai-product-use-cases

tothmano committed:
Update product-analytics.mdx

tothmano committed:
Merge branch 'mano/ai-product-use-cases' of https://github.com/axiomhq/docs into mano/ai-product-use-cases

tothmano committed:
Update vector.mdx

tothmano committed:
Merge branch 'main' into mano/ai-product-use-cases

manototh commented:
Drop in favour of https://github.com/axiomhq/docs/pull/408
